api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -352,22 +360,14 @@ public Properties getRuntimeProperties() {
 		// we don't want to try to load core modules in tests
 		runtimeProperties.setProperty(ModuleConstants.IGNORE_CORE_MODULES_PROPERTY, "true");
 		
-		try {
-			File tempappdir = File.createTempFile("appdir-for-unit-tests-", "");
-			tempappdir.delete(); // so we can make it into a directory
-			tempappdir.mkdir(); // turn it into a directory
-			tempappdir.deleteOnExit(); // clean up when we're done with tests
-			
-			runtimeProperties.setProperty(OpenmrsConstants.APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY, tempappdir
-			        .getAbsolutePath());
-			OpenmrsUtil.setApplicationDataDirectory(tempappdir.getAbsolutePath());
-		}
-		catch (IOException e) {
-			log.error("Unable to create temp dir", e);
-		}
-		
-		return runtimeProperties;
-	}
+		// we don't want to try to load core modules in tests
```

Comment:
you have duplicated 2 lines
and please fix the indentation. now that the try catch is removed we need one less tab.